.changeset/wise-plums-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": minor
```

Comment:
Sounds like a patch would be more suitable?

Reply:
```suggestion
"@kaizen/components": patch
```